models/tacotron.py

```diff
@@ -28,6 +29,9 @@ def __init__(self,
         self.linear_dim = linear_dim
         self.embedding = nn.Embedding(num_chars, 256)
         self.embedding.weight.data.normal_(0, 0.3)
+        if num_speakers > 0:
```

Comment:
This makes more sense to be num_speakers > 1. Default setting is already num_speaker = 1

Reply:
yeah that's a fair point